LibOS/shim/src/shim_checkpoint.c

```diff
@@ -803,7 +806,7 @@ int receive_handles_on_stream (struct palhdl_header * hdr, ptr_t base,
 
 static void * cp_alloc (struct shim_cp_store * store, void * addr, int size)
 {
-    int flags = MAP_PRIVATE|MAP_ANONYMOUS|VMA_INTERNAL;
+    int flags = MAP_PRIVATE|MAP_ANONYMOUS|VMA_INTERNAL|VMA_CP;
```

Comment:
Please use the defined constant (`CP_FLAGS`).

Reply:
I don't think it will make any difference because the compiler is perfectly capable of optimizing this code.

Reply:
Yes, of course it will get compiled exactly the same way (it's a define), but this wasn't my concern at all ;)

My point was to use the defined constant for consistency and readability. It seems that you added a defined constant for this set of flags, but you use it only sometimes. IMO it's error-prone, because someone can later update the flags only in the define and forget to fix them here.

Nonetheless, seems that this issue is already fixed in the newest version, thanks :)